src/main.ts

```diff
@@ -215,6 +215,7 @@ function isSupportedPlatform(platform: PlatformInformation): boolean {
     }
 
     if (platform.isLinux()) {
+        // TODO: Add 'arm' and 'arm64'
```

Comment:
Returning false here means that we cannot even debug. Is that what we want to do? Or do we want to return true here, but be smart enough to not start the language server and output a big comment to the log indicating why we are doing that?

Reply:
Idealy, we would activate when the whole extension is ready to go so users get the full extension feature in VS Code.

@JoeRobich Do you know if the language service would be ready for Linux ARM/ARM64 soon or should only debugger activate for now?